lib/google/cloud/gemserver/cli/server.rb

```diff
@@ -170,6 +168,29 @@ def extract_key response
           end
 
           ##
+          # @private Creates a Gemfile and Gemfile.lock for the gemserver that
+          # runs on Google App Engine such that gemstash is not required
+          # client side for the CLI.
+          def gemfile
+            source = %{
+source "https://rubygems.org"
+
+gem "concurrent-ruby", require: "concurrent"
+gem "gemstash", git: "https://github.com/bundler/gemstash.git"
```

Comment:
I think we should pin to a specific known good commit as long as we're downloading from github. Using HEAD is risky.